### PR TITLE
clickhouse-odbc: revision bump (icu4c 73.2)

### PR DESCRIPTION
**Pull request by carlocab**
See #133611.

### CODE REVIEW

**Comment by carlocab** [comment]
This is using a really old vendored version of `folly`: https://github.com/facebook/folly/tree/b817015da18c9f121f376d033fa47450651cd6ef

**Comment by carlocab** [comment]
Let's see if this fails on `master`: https://github.com/Homebrew/homebrew-core/actions/runs/5273552480

**Comment by carlocab** [comment]
Ok, so this builds on `master`. Weird.

**Comment by carlocab** [comment]
Ah, got it. The issue is, indeed, their old vendored copy of `folly`: https://github.com/facebook/folly/issues/1867

File: Formula/clickhouse-odbc.rb [hunk 1]
```diff
@@ -14,15 +14,16 @@ class ClickhouseOdbc < Formula
   end
 
   bottle do
-    sha256 cellar: :any,                 arm64_ventura:  "9861a2a1c4cfd5e91da5bac18199324b960daaebd6d6c2cd63641a51124fa80e"
     sha256 cellar: :any,                 arm64_monterey: "7db18ffe67c8d28059a4bfd6835f053fd48a6c12a21ad385df2aabf84ab87e60"
     sha256 cellar: :any,                 arm64_big_sur:  "8178a93f509e725ba63e318059851183ed8d599c6a9eda6a1918258f63400ab5"
-    sha256 cellar: :any,                 ventura:        "00c541af823af303db5f25af6eed62ed43410b6f0a8d5459b72dfc33eef408a2"
```
Comment (carlocab) [inline Formula/clickhouse-odbc.rb:20]: Ventura bottles will fail to build, hence, deprecated. See below.

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**